src/main/java/org/folio/notes/service/impl/ConfigurationServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.notes.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import org.folio.notes.client.ConfigurationClient;
+import org.folio.notes.client.ConfigurationClient.Configuration;
+import org.folio.notes.client.ConfigurationClient.ConfigurationCollection;
+import org.folio.notes.service.ConfigurationService;
+
+@Slf4j
```

Comment:
I don't see where you use logger. If you didn't want to log smth, then, please, remove the annotation.